rust/agama-server/src/l10n/model/locale.rs

```diff
@@ -106,11 +113,20 @@ impl LocalesDatabase {
                 .or_else(|| names.name_for(DEFAULT_LANG))
                 .unwrap_or(territory.id.to_string());
 
+            let consolefont = match language.consolefonts.consolefont.first() {
```

Comment:
In those cases where you are not going to do anything with the `None`, I would use a `map`:

```rust
let consolefont = language
    .consolefonts
    .consolefont
    .first()
    .map(|f| f.id.clone());
```

It looks longer just because of the format.

PS: Out of the scope of this PR, we might consider in the future to implement the iterator for the `RankedConsoleFonts.